recipes/libosmium/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,13 @@
+#include <osmium/io/any_input.hpp>
+
+#include <iostream>
+
+int main() {
+    try {
+        const osmium::io::File input_file{"missing_file.pbf"};
```

Comment:
Nice idea with passing a missing file, very on topic with the simplification of the test packages :)